cassandra/cassandra-docker.sh

```diff
@@ -130,6 +130,51 @@ if [ -n "$HELP" ]; then
   exit 0
 fi
 
+if [ -z "${MAX_HEAP_SIZE}" ] && [ -z "$MEMORY_LIMIT" ]; then
+  echo "The MAX_HEAP_SIZE and MEMORY_LIMIT envars are not set. Defaulting the MAX_HEAP_SIZE to 512M"
+  export MAX_HEAP_SIZE=512M
```

Comment:
The reason I had this here was to fall back to the old behaviour we had before (which was to use 512MB).

I will update this to use the cgroups file instead as I believe that will always be available and its a good fallback.